.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: Test
+on: [pull_request]
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.12.x, 1.13.x]
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Checkout code
+        uses: actions/checkout@v1
+
+      - name: Cache modules
+        uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Download modules
```

Comment:
I guess this step is not necessary because modules get installed by go test command.

Reply:
My understanding is that `-i` option is necessary if you want to download dependencies:

```
go test -i
```

, and test itself is not executed with this option.

But if what you say is true, the config can be more simple.
Could you please give me some documents describing it.

Reply:
> Standard commands like go build or go test will automatically add new dependencies as needed to satisfy imports (updating go.mod and downloading the new dependencies).

https://github.com/golang/go/wiki/Modules#daily-workflow